lib/passport-azure-ad/wsfedstrategy.js

```diff
@@ -51,7 +53,7 @@ function Strategy (options, verify) {
   this.certs = [];
 
   // Create the metadata object if the user has specified a federation metadata url
-  if(options.identityMetadata) {
+  if (options.identityMetadata) {
     this.metadata = new Metadata(options.identityMetadata);
```

Comment:
Always best to open an issue, as this will get lost.
https://github.com/AzureAD/passport-azure-ad/issues/35